src/JobPanel/backend/service_proxy.py

```diff
@@ -67,29 +72,35 @@ def __init__(self, repeater, service_data, connection):
     def call(self, method_name, data, result):
         """
         Call 'method_name(data)' of the remote service returned result is appended
-        to the result.
+        to the result which has to be a list.
+
+        TODO: If necessary implement alternative method which calls a given answer processing once the  answer is received.
+        TODO: How we process errors. !!!!
+
+
         :param method_name: Name of method to call.
         :param data: Serializable (JSONData) passed to the method.
         :param result: List of results.
-        :return:
+        :return: None, error.
         """
+
+        # see: ServiceBase.save_result
+        self.repeater.send_request(..., {'action': 'save_result', 'data': result})
         pass
 
     def start_service(self):
         """
-        Process of starrting a Job
-        1. # have service configuration data (from constructor)
-        2. # get connection parameter - (in constructor)
-        3. upload job files (specified in service_data, using the connection, upload config file
-        4. get delegator proxy from connection
-        5. setup remote port forwarding for the starter port (get it from repeater)
-
-        6. repeater.expected_answer ... Sort of implicit request, we expect that ClientDispatcher
-           form an answer when child service connection is accepted and we recieve its listenning port.
-           We set that connect_service should be called `on_answer`.
-
-        7. submit (or start) job through delegator
-        8. set  Job state as 'queued'
+        Process of starting a Job:
+        0. # have repeater, connection, and service configuration data (from constructor)
+        1. upload job files (specified in service_data, using the connection, upload config file).
```

Comment:
Každé spuštění analysis odpovídá jednomu spuštění MJ a má vlastní podadresář ve workspace, uvnitř něj jsou pak podadresáře pro spuštění Jobů. Tato struktura nemusí být kompletní na všech počítačích, ale na konci je možno vše stáhnout na Root. Config pro backend by mohl byt primo ve workspace. Configy pro MJ a Joby budou v jejich adresarich. Predpokladam, ze to tak ma i Pavel. Zeptej se ho, ve stredu by mel byt.